### ISSUE TITLE
Chord Symbols do not add to palettes properly (TPC is missing)

### ISSUE THREAD

**Issue by worldwideweary** [musescore/MuseScore#18333]
### Issue type

Other type of issue

### Steps to reproduce

1. New Score
2. Add a Chord Symbol, for example Cmaj7
3. Add symbol to palette (Ctrl+Shift+Drag)
Result: instead of showing Cmaj7 on the cell information of the palette, the result is "maj7"

This was also the case in 3.x behavior. Some how the TPC information is omitted in the process. 

Adding chord symbols to a palette doesn't seem to be very common based on forum use over the years, but the idea is to be able to make palette of common chords to enable placing them onto a score via mouse as an alternative to Chord entry with the keyboard manually. I've got a custom 3.x build that does this and automatically selects the next ChordRest afterwards to facilitate placement of more symbols. Niche case, I presume, but it might be worth at least fixing the view of the chord cell on a palette to show the roots.

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/7139517/be1a1d6e-f1ea-4c7a-b5e5-68b8891f5c0e

Niche case of continual entry is what I have the ability to do, and maybe that might be considered worth implementing in 4? A quick example is given:

https://github.com/musescore/MuseScore/assets/7139517/3f052054-253b-48c7-b727-60b680db1072

It's nice not having to type it in given a reasonable palette setup.

### MuseScore Version

4.1 Beta

### Regression

No.

### Operating system

Linux XFCE

### Additional context

As a pointer, in 3.6 code the Harmony::Render function is used to concoct the display of a chord symbol.
When producing into a palette cell, apparently RenderActionType::Note/Accidental is never hit but only SET...

 if (a.type == RenderAction::RenderActionType::SET) {

**Comment by MarcSabatella** [comment]
Unfortunately, the way the palette stores elements is not really compatible with the way chord symbols are represented and rendered, especially when you consider transposition.  In theory it might be possible to hack something together, but anyhow, there's definitely a reason this has never been supported.

**Comment by worldwideweary** [comment]
@MarcSabatella: Yeah, and it figures — had to do some refactoring in the harmony render functions to get MS3.x to perform acceptably well in the 3.x code to get palette results like the following screenshot:

![palettesChord](https://github.com/musescore/MuseScore/assets/7139517/ce2631b3-aa16-47c8-9a2b-349b8b9b3929)


Still, it *is* nifty to be able to quickly insert chord symbols using palette items. Would be nice to see it happen in the future for MS4 users

**Comment by bkunda** [comment]
@DmitryArefiev please review when you have the chance
